hw/ip/otp_ctrl/data/earlgrey_a0_skus/sival/BUILD

```diff
@@ -37,30 +79,68 @@ cc_library(
     ],
 )
 
-MISSION_MODE_OVERLAYS = [
-    "//hw/ip/otp_ctrl/data:otp_json_hw_cfg",
-    "//hw/ip/otp_ctrl/data:otp_json_fixed_secret0",
-    "//hw/ip/otp_ctrl/data:otp_json_secret1",
-    "//hw/ip/otp_ctrl/data:otp_json_fixed_secret2",
-    "//hw/ip/otp_ctrl/data/earlgrey_a0_skus/generic:alert_digest_cfg",
-    "//hw/ip/otp_ctrl/data/earlgrey_a0_skus/generic:otp_json_creator_sw_cfg",
-    "//hw/ip/otp_ctrl/data/earlgrey_a0_skus/generic:otp_json_owner_sw_cfg",
-]
-
+# Initial test_unlocked state. Only applicable for test_unlocked0. All other
+# test states required the SECRET0 partition to be configured.
+# In this configuration, ROM execution is disabled by default. JTAG should be
+# used to bootstrap code into SRAM or FLASH.
 otp_image(
-    name = "otp_img_prod_personalized",
-    src = "//hw/ip/otp_ctrl/data:otp_json_prod",
-    overlays = MISSION_MODE_OVERLAYS,
+    name = "otp_img_test_unlocked0_manuf_empty",
+    src = "//hw/ip/otp_ctrl/data:otp_json_test_unlocked0",
+    overlays = [
+        "//hw/ip/otp_ctrl/data:otp_json_exec_disabled",
+    ],
+    visibility = ["//visibility:private"],
 )
 
+# `MANUF_INITIALIZED` configuration. This configuration will be generally used
+# to lock the chips before shipping to the Final-Test test house.
 otp_image(
-    name = "otp_img_prod_end_personalized",
-    src = "//hw/ip/otp_ctrl/data:otp_json_prod_end",
-    overlays = MISSION_MODE_OVERLAYS,
+    name = "otp_img_post_cp_test_locked0",
+    src = "//hw/ip/otp_ctrl/data:otp_json_test_locked0",
+    overlays = MANUF_INITIALIZED,
 )
 
-otp_image(
-    name = "otp_img_dev_personalized",
-    src = "//hw/ip/otp_ctrl/data:otp_json_dev",
-    overlays = MISSION_MODE_OVERLAYS,
-)
+# `MANUF_INITIALIZED` OTP configuration. Available on TEST_UNLOCK states 1-7.
+[
+    otp_image(
+        name = "otp_img_test_unlocked{}_manuf_initialized".format(i),
+        src = "//hw/ip/otp_ctrl/data:otp_json_test_unlocked{}".format(i),
+        overlays = MANUF_INITIALIZED,
+    )
+    for i in range(1, 8)
```

Comment:
Our current provisioning scheme only makes use of TEST_UNLOCKED{0,1}, so we could limit this further, thoughts?

Reply:
Let's leave it as is for now. I am not sure yet if there is going to be a need for additional test insertions across different sites.